packages/deployments/contracts/src/tasks/preflight.ts

```diff
@@ -174,5 +178,15 @@ export default task("preflight", "Ensure correct setup for e2e demo with a speci
       } else {
         console.log("*** Sufficient relayer fees present!");
       }
+
+      // Add relayer
+      const relayer = _relayer ?? process.env.RELAYER_ADDRESS;
+      if (relayer && isAddress(relayer)) {
```

Comment:
should error if it is not an address

Reply:
https://github.com/connext/nxtp/commit/6caf990bba55cce35f08d56ef5ecf378bcb872cf